tests/system/Router/RouteCollectionTest.php

```diff
@@ -1744,6 +1745,46 @@ public function testRouteOverwritingMatchingHost(): void
         $this->assertSame($expects, $router->handle('/'));
     }
 
+    public function testRouteMatchingHostMultipleCorrect(): void
+    {
+        service('superglobals')->setServer('HTTP_HOST', 'two.domain.com');
+        service('request')->setMethod(Method::GET);
+
+        $routes = $this->getCollector();
+        $router = new Router($routes, Services::request());
+
+        $routes->setDefaultNamespace('App\Controllers');
+        $routes->setDefaultController('Home');
+        $routes->setDefaultMethod('index');
+
+        $routes->get('/', 'Home::index', ['as' => 'ddd']);
+        $routes->get('/', '\App\Controllers\Site\CDoc::index', ['hostname' => ['one.domain.com', 'two.domain.com', 'three.domain.com']]);
+
+        $expects = '\App\Controllers\Site\CDoc';
```

Comment:
Please change to `$expect` because it is a single value.

Reply:
Done